code/modules/reagents/reagent_dispenser.dm

```diff
@@ -23,10 +23,10 @@
 	else
 		return ..()
 
-/obj/structure/reagent_dispensers/New()
+/obj/structure/reagent_dispensers/Initialize()
 	create_reagents(tank_volume)
 	reagents.add_reagent(reagent_id, tank_volume)
-	..()

```

Comment:
This was running code before parent New() that would now happen after New().

Does this matter in this case?